topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.md

```diff
@@ -184,29 +189,30 @@ In this tutorial, we will use BlobToolKit in order to integrate the following da
 
 In the next steps, we will generate the data required for generating the visualization plots with BlobToolKit.
 
-## Generate read coverage data with **HISAT2**
+## Generate read coverage data with **Minimap2**
 
 Read coverage is an essential metric for evaluating the quality of genome assemblies, and it provides valuable information for identifying regions of high and low quality, detecting misassemblies, and identifying potential contaminants. Thus, for example, unexpected regions of low coverage suggests potential errors, such as misassemblies, gaps, or low complexity regions ({% cite Koren2017 %}).
 
-In this tutorial we will use **HISAT2** for generation the coverage data. This tool uses a indexing scheme based on the Burrows-Wheeler transform (BWT) and the Ferragina-Manzini (FM) index, which enables efficient and accurate alignment ({% cite Zhang2021 %}). It then provides the alignment output in BAM file format which we will then use as an input for BlobToolKit.
+In this tutorial we will use **Minimap2** for generation the coverage data. Minimap2 is a multi-purpose aligner which is particularly efficient at aligning long-reads produced by sequencing machines from PacBio or Oxford Nanopore ({% cite Li2018 %}).
 
 > <comment-title>How is coverage information encoded in the BAM file?</comment-title>
 >
 > Coverage information is encoded in the BAM file format through the number of reads that align to each position in the reference genome. Specifically, the BAM file stores each read's start and end position in the genome, as well as the length and orientation of the read, among other information. To calculate coverage, the number of reads that overlap each position in the reference genome is counted.
 >
 {: .comment}
 
-> <hands-on-title>Generate BAM file with HISAT2</hands-on-title>
+> <hands-on-title>Generate BAM file with Minimap2</hands-on-title>
 >
 > 1. {% tool [Collapse Collection](toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0) %} with the following parameters:
 >    - {% icon param-collection %} *"Collection of files to collapse into single dataset"*: `CReniformis_Pacbio`
 >
 >
-> 2. {% tool [HISAT2](toolshed.g2.bx.psu.edu/repos/iuc/hisat2/hisat2/2.2.1+galaxy1) %} with the following parameters:
->    - *"Source for the reference genome"*: `Use a genome from history`
+> 2. {% tool [Map with minimap2](toolshed.g2.bx.psu.edu/view/iuc/minimap2/5cc34c3f440d) %} with the following parameters:
```

Comment:
This ID looks wrong. Did you mean
```suggestion
> 2. {% tool [Map with minimap2](toolshed.g2.bx.psu.edu/repos/iuc/minimap2/minimap2/2.28+galaxy0) %} with the following parameters:
```

You can get this ID from the Galaxy tool from, top bar, arrow-down, tool-ID.